bleak/backends/scanner.py

```diff
@@ -111,3 +111,34 @@ async def _find_device_by_address(
             await self.stop()
 
         return device
+
+
+class AdvertisementData:
+    """
+    Wrapper around the advertisement data that each platform returns upon discovery
+    """
```

Comment:
Would be nice to list the allowable kwags in the docstring.

Reply:
Updated to include required and keyword args